crates/teloxide-core/src/net.rs

```diff
@@ -70,10 +72,13 @@ pub fn client_from_env() -> reqwest::Client {
 ///
 /// [issue 223]: https://github.com/teloxide/teloxide/issues/223
 pub fn default_reqwest_settings() -> reqwest::ClientBuilder {
-    reqwest::Client::builder()
+    #[cfg(not(target_family="wasm"))]
+    return reqwest::Client::builder()
         .connect_timeout(Duration::from_secs(5))
         .timeout(Duration::from_secs(17))
-        .tcp_nodelay(true)
+        .tcp_nodelay(true);
+    #[cfg(target_family="wasm")]
+    return reqwest::Client::builder();
 }
```

Comment:
Why?

This straight up can't work with teloxide, we need a long timeout.

Reply:
What is this timeout for? Long polling?

If we need long polling in browser I think we need this, and I think the primary usage is Workers and it accepts webhook. I have not found other APIs needs long timeout.

If possible I can try to find how we can set timeout in wasm reqwest